src/components/Preferences/CoursePreferencesTable/index.tsx

```diff
@@ -55,9 +55,6 @@ const CoursesPreferencesTable = ({ values, setFieldValue, isDisabled }) => {
                         key={c + "-willingness"}
                         isDisabled={isDisabled}
```

Comment:
Did you add the not qualified? Or just remove no selection

Reply:
Yeah it should just be renamed

Reply:
I misunderstood the requirement. Will fix